csharp4python/classes-and-objects-encapsulating-data/README.md

```diff
@@ -185,21 +185,21 @@ Here are some examples to illustrate. Each class would be in its own file, but w
 ```csharp
 public class FortyTwo
 {
-    public int IntValue { get; set; } = 42;
+    public int intValue = 42;
```

Comment:
Changed this to a field, as I believe that was the intent of the demo--to contrast this non-`readonly` field to other fields that *are* `readonly`.